core/src/main/scala/org/apache/spark/internal/config/package.scala

```diff
@@ -597,6 +611,53 @@ package object config {
       .toSequence
       .createWithDefault(Nil)
 
+  private[spark] val UI_ENABLED = ConfigBuilder("spark.ui.enabled")
```

Comment:
Could you put UI-related options in a new file (e.g. `UI.scala`)?

I was going to suggest `WebUI.scala` but that already exists (albeit in a different place).

Reply:
Thanks for the nice suggestion. Will address.